source/mainnet/net/release-notes/release-notes-lp.rst

```diff
@@ -288,13 +288,23 @@ Wallets
 |bw|
 -------------------------
 
-    April 4, 2024
+    June 20, 2024
 
-    Version 1.5.1 fixes a bug related to the injected script being loaded multiple times which was causing performance issues on the browser.
+    Version 1.5.2 includes several changes and bug fixes.
+        -   Added new option to edit account name. Name saved in local storage. Changed name displayed across all BrowserWallet.
+        -   Additional error message. Now instead of not showing invalid tokens, they displayed in token list with corresponding error. In order to show, that we found tokens in contract, but they have error.
+        -   Display the optional name for ID providers when present, otherwise fallback to the chain name.
+        -   Increased padding for QR code background. In dark mode, QR code not blending with background.
```

Comment:
The second sentence could refer to the reason for the change (this seems to be the case here) or describe the state after the change. I suggest removing the second sentence since it might not be that important to know the exact reason (the reason for a small styling change is likely that it looked weird before).

```suggestion
        -   Increased padding of the QR code background element.
```